tm2/pkg/bft/node/node.go

```diff
@@ -843,11 +844,9 @@ func makeNodeInfo(
 	genDoc *types.GenesisDoc,
 	state sm.State,
 ) (p2p.NodeInfo, error) {
-	txIndexerStatus := "on"
-	if _, ok := txIndexer.(*null.TxIndex); ok {
-		txIndexerStatus = "off"
-	} else if txIndexer == nil {
-		txIndexerStatus = "none"
+	txIndexerStatus := p2p.IndexerStatusOff
```

Comment:
p2p shouldn't be aware of indexers.
p2p is a general p2p library.
indexer status should be in the index (well, should be named like "eventstore" I think) package.

Reply:
Moved to the `eventstore` package:
[20e9a5c](https://github.com/gnolang/gno/pull/546/commits/20e9a5c2b98ff97f800b494af6c2492f11a2c6ea)